src/style/home.css

```diff
@@ -123,8 +128,9 @@ header nav ul li:hover .single-nav {
   }
 
   .main-content {
-    width: 80%;
+    width: calc(100vw - 150px);
     position: absolute;
-    left: 15%;
+    left: 150px;
+    padding-top: 0;
   }
 }
```

Comment:
@byiringiroscar I change this previous values . You need to change values again with git conflicts